### ISSUE TITLE
Why iota lost?! Scam?!

### ISSUE THREAD

**Issue by mazyartrana** [iotaledger/wallet#552]
I've upgraded my iota wallet to v2.5.4 and my balance is zero !! while I had 3000 iota at august 2017.

I searched over internet and found two solutions :

   1. click attach to tangle over and over again
   2. use reclaim tool

I tested both methods but my balance is 0.

I funded my iota wallet at 19 august through bitfinex and when I see the blockchain. I can see money moved from my address to another address on 23 August !! I am not familiar with iota blockchain, is it natural?

here is the link that show my address

https://iotasear.ch/address/QYGBNJVBVEYDQUEWBMLVDCEBRCKWDQILKYQKSQZACZPQYUKQGMXCPWGMVPPRAPLAPLNELURTIUVKYUISDNDMUSJLXW

Please help me how can I recover my iota!! I didn't make any transfer and when I get login, my balance is 0. I see too many people has same problem! so why you support this project? its look like scam!

**Comment by carloss42** [comment]
Hi, it looks like someone transferred out your Iota 3 days after your deposit. How did you make the seed? In August there were a lot of scam seed generators that were unknown

**Comment by mazyartrana** [comment]
I generate iota seed with windows powershell, I didn't use third party website to generate it ! also I have more coins on same PC that doesn't move anywhere. how is possible that I hacked?!

**Comment by wonkytonky** [comment]
windows power shell is insecure it doenst generate true random seeds

**Comment by mazyartrana** [comment]
Really?!!!! Damn it ! Why developers of IOTA don't created a seed generator in the wallet ! online seed generator websites are scam ! powershell is not secure ! they could create a seed generator with a few lines of code or even write a guide line to generate seed on the correct way.

I don't believe this project anymore! only scammers create such a big security hole on their system.

**Comment by mazyartrana** [comment]
Lesson learned. It cost me 3000 bucks. bye  fu....ing iota.

**Comment by carloss42** [comment]
yes it was announced that generating a seed through powershell is not secure. sorry for your loss

**Comment by mazyartrana** [comment]
This is not for everyone who download wallet. a newbie search in google "generate iota wallet seed" not about powershell security. who knows windows powershell is insecure!? this is deliberate hole.

**Comment by wonkytonky** [comment]
the wallet used to have a seed generator in it. but it's wise to not do this.. just imagine the seed generator of the wallet for some reason is not secure  anymore . then all seed are in danger. so they leave the seed generation  management over to the ppl themselves. 

unfortunately a lot of  ppl giving wrong advice about seed generation.    

best way is to  trow some alphabet dice  or use a good randomizer.. you trust.   

i use keepass  with added entropy and I manual change a few letters. 
then story my seed encrypted ii keypass on a few usb drives that are offline. 

don't trust a website that gives a seedgenerator, or advice.. a lot of scammers trying to get seeds of non tech savy users.

**Comment by Webbanditten** [comment]
@sl00k Waw... Maybe people dont want to take a Ph.D in crypto, to create a freaking wallet. The easier it is to have people joining your blockchain the more people will be allowed to join.

**Comment by mathiasrw** [comment]
This will help you: https://forum.helloiota.com/588/Help-My-IOTA-balance-is-zero-steps-to-help-you-find-your-balance-v254?PageIndex=1

**Comment by Craigcal56** [comment]
THE WHOLE SHOW SUCKS THE WALLET THE NETWORK A BIG PIPE DREAM BIT OFF WAY TO MUCH   i QUITE WHILE AHEAD HERE   WENT TO EOS AND WAVES much better  AINT GOT TO CHASE YOUR  FUCKING COINS AROUND THE NETWORK AT eos or waves AND THERE BOTH TAKING OFF AS WE SPEAK

**Comment by hernancmartinez** [comment]
@wonkytonky when you said

> windows power shell is insecure it doenst generate true random seeds
> i dont fine your addresss https://ipfs.io/ipfs/Qmeb4B5AEi5MWTreKPRtufBFRFWRVTpZkBpX6FeaXCAe3B here so it looks like funds are taken out by someone else

What is that list and what does it mean to be on that list?

**Comment by wonkytonky** [comment]
that list is iota foundation who  took money away form insecure wallets due to key reuse  which would make it easy for hackers to steal the money.. so they moved the money somewhere safe and you can reclaim it.      the claim process is happening today so ppl are getting their money back.

**Comment by Jared-1999** [comment]
I transferred miota from my binance account to my iota ledger wallet 3 days ago and its still not showing up in my wallet or in the history section of the wallet. can someone help please?

**Comment by bennyxvi** [comment]
Have you lost money online? It is possible to recover all you have lost using computer intelligence and recovery expert. I have used the service of a hacker to recover money lost with bitcoins. You can contact him too cybertronxiii@gmail.com

**Comment by hello1234567894** [comment]
I am having the same problem..have you resolved the issue? I emailed customer support 2 weeks ago and have had no response. What do I do?
